pages/osx/dd.md

```diff
@@ -5,19 +5,19 @@
 
 - Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`) and show the progress:
 
-`dd if={{path/to/file.iso}} of={{/dev/usb_drive}} status=progress`
+`dd if={{path/to/file.iso}} of={{/dev/usb_device}} status=progress`
 
 - Clone a drive to another drive with 4 MB block, ignore error and show the progress:
 
-`dd if={{/dev/source_drive}} of={{/dev/dest_drive}} bs={{4m}} conv={{noerror}} status=progress`
+`dd if={{/dev/source_device}} of={{/dev/dest_device}} bs={{4}}m conv={{noerror}} status=progress`
```

Comment:
I prefer `{{4m}}`  because `m` is not a mandatory unit for `bs`, it can be 4k, 512, and so on. 
```suggestion
`dd if={{/dev/source_device}} of={{/dev/dest_device}} bs={{4m}} conv={{noerror}} status=progress`
```

Reply:
Makes sence even it will not be convertible placeholder.